gui/default/syncthing/core/syncthingController.js

```diff
@@ -1847,7 +1847,7 @@ angular.module('syncthing.core')
                     }
                 };
                 delete folderCfg.trashcanFileVersioning;
-                delete folderCfg.trashcanClean;
```

Comment:
Deleting this attribute is proper, because it's a temporary one just used by the config editor and shouldn't be in the config that we post to the backend.

We could make the folder editor use a different temporary object, or maybe delete this after the modal has closed but before posting, or maybe use [setPristine](https://docs.angularjs.org/api/ng/type/form.FormController#$setPristine) to disable the validations while the dialog is closing.

Reply:
All right, I'll do it again some other way.